source/menus/inventory_menu.cpp

```diff
@@ -10,8 +10,8 @@ Line lines[LINE_NUM] = {
     {"amounts", AMOUNT_INDEX, "Modify amounts for health, magic, etc."},
     {"item inventory", ITEM_INVENTORY_INDEX, "Modify item inventory menu"},
     {"quest status", QUEST_STATUS_INDEX, "Modify quest status menu"},
-    {"spoils", SPOILS_INDEX, "Modify spoils bag items/amounts (coming soon...)"},
     {"delivery bag", DELIVERY_BAG_INDEX, "Modify delivery bag items/amounts (coming soon...)"},
+    {"spoils", SPOILS_INDEX, "Modify spoils bag items/amounts (coming soon...)"},

```

Comment:
Should delete the "coming soon..." from the menu descriptions now that the features are coming now